cmd/support-logging/main.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/edgexfoundry/edgex-go/internal/pkg/usage"
 	"github.com/edgexfoundry/edgex-go/support/logging"
 	"github.com/edgexfoundry/edgex-go/support/logging-client"
+	"strconv"
```

Comment:
Since strconv is a package from the std library, please include it with the other std library packages above in alphabetical order.